src/collections/_documentation/platforms/javascript/react.md

```diff
@@ -146,3 +146,134 @@ class App extends React.Component {
 
 export default App;
 ```
+
+## Redux
+
+{% version_added 5.20.0 %}
+
+Redux support is included in the `@sentry/react` package since version `5.20.0`. To apply Sentry to Redux, use `Sentry.createReduxEnhancer` at the same place that you initialize your Redux store.
+
+
+```js
+import { createStore, compose } from 'redux';
+import * as Sentry from '@sentry/react'
+
+// ...
+
+const sentryReduxEnhancer = Sentry.createReduxEnhancer({
+  // Optionally pass options listed below
+});
+
+const store = createStore(rootReducer, sentryReduxEnhancer);
+
+// ...
+
+```
+
+If you have other enhancers or middleware such as `thunk`:
+
+```js
+const store = createStore(rootReducer, compose(applyMiddleware(thunk), sentryReduxEnhancer));
+```
+
+{% capture __alert_content -%}
+Sentry uses a redux **enhancer**. Pass the enhancer, as indicated above. Don't pass it to `applyMiddleware` and don't call the method when you pass it to the `createStore` method.
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Note"
+  content=__alert_content
+  level="info"
+%}
+
+### Normalization Depth
+
+By default Sentry SDKs normalize any context to a depth of 3, which in the case of sending Redux state you probably will want to increase that. You do so by passing `normalizeDepth` to the `Sentry.init` call:
```

Comment:
> which in the case of sending Redux state you probably will want to increase that.

This sentence sounds kind odd, but I might be wrong. Can you take a look at it @MimiDumpling ?